www/js/app.js

```diff
@@ -145,6 +145,32 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('searchArticles').click();
         return false;
     });
+     //Handle Home key press to focus #prefix
+     $(window,document).on('keydown', function(e) {
+         //if home key is pressed then
+         if(/^Home/.test(e.key)) {
```

Comment:
Is there a specific reason to test that `e.key` **starts** with `Home`?
If not, it would be safer to test for equality : `/^Home$/`

Reply:
changed to equality
 `if(key === 'Home')`